concent_api/core/message_handlers.py

```diff
@@ -369,13 +369,11 @@ def handle_send_force_get_task_result(client_message: message.concents.ForceGetT
         task_to_compute,
         requestor_public_key,
     )
+    maximum_download_time = calculate_maximum_download_time(client_message.report_computed_task.size, settings.MINIMUM_UPLOAD_RATE, )
```

Comment:
Break 
parameters
into 
separate
lines.

Reply:
done

Reply:
I think it was fine (except for the stray comma at the end). It was not that long, it did not have to get broken into multiple lines.